doc/whatsnew/fragments/7569.bugfix

```diff
@@ -0,0 +1,3 @@
+Use `py-version` to determine if a message should be emitted for messages defined with `max-version` or `min-version`.
```

Comment:
Single backticks just create italics, so for code highlighting, use two.

Reply:
```suggestion
Use ``py-version`` to determine if a message should be emitted for messages defined with ``max-version`` or ``min-version``.
```